lib/src/presentation/widgets/movie_list_page_widgets/now_playing_movies_page_view.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:core';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_project/src/core/utils/extensions.dart';
+import '../../../core/utils/styles.dart';
+import '../../../domain/entities/genre.dart';
+import '../../../domain/entities/movie.dart';
+import '../../bloc/movie_list_page_bloc.dart';
+import '../../pages/detail_movie_page.dart';
+
+class nowPlayingMoviesPageView extends StatelessWidget {
+  const nowPlayingMoviesPageView({
+    super.key,
+    required this.movies,
+    required this.allGenres,
+    required this.bloc,
+  });
+
+  final MovieListPageBloc bloc;
+  final List<Movie> movies;
+  final List<Genre> allGenres;
+
+  static const String nowPlayingTextTitle = "Now Playing";
+  static const Key gestureDetectorKey = Key('PageViewNavigateToDetails');
+
+  static const double nowPlayingTitleFontSize = 30;
+  static const double pageViewSizeBoxHeight = 700;
+
+  static const double containerPageViewTextHeight = 100;
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        PageView(
+          scrollDirection: Axis.horizontal,
+          children: [
+            for (int movieNumber = 0;
+                movieNumber < movies.length;
+                movieNumber++)
+              GestureDetector(
+                key: gestureDetectorKey,
```

Comment:
Beware of having the same key for multiple widgets

Reply:
I use it only for the navigation test, it's a bad use of it?

Reply:
You shouldn't have the same key for multiple widgets. it'd be better to either make your unit test not dependant on those keys, or define a logic for the key. For example, that the key is compiled as `Key('PageViewNavigateToDetails$index')`